app/views/admin_notice_mailer/notice_for_councillor_contribution.html.haml

```diff
@@ -0,0 +1,6 @@
+%p
+  Hi Admin,
+
+%p
+  There is a new submission of councillor contribution. Please check the
```

Comment:
I think this should link directly to the contribution admin show view, rather than the index. So it's a bit clearer what's actually come in.

It would also be nice to have a bit of info about it in the email, like:

> Hi there humans,
> 
> You've got a wonderful [new councillor contribution from anonymous for Bayside Council](link).
> You can view and export the contribution in the PlanningAlerts Admin area:
> 
> link
> 
> Don't forget to thank the lovely human who made this contribution!
> 
> Have a great day,
> 
> PlanningAlerts

Reply:
@equivalentideas I update the haml as 
>Hi There humans,

>You've got a wonderful new councillor contribution in Marrickville Council. Here is the[link.](link).

>Don't forget to thank the lovely human who made this contribution if you know who they are!

>Have a great day,

>PlanningAlerts

I did not include the name of the contributor, because as we have the `AdminNoticeMailer` under `create` method of the councillor contribution, we don't have the information of the contributor at this point.